src/styles/components/_component-grid.scss

```diff
@@ -0,0 +1,72 @@
+$breakpoint-tablet: 820px;
+
+.components-grid {
+  width: 100%;
+  border: 1px black solid;
+  text-align: center;
+  padding: 1rem;
+
```

Comment:
This might eventually be handled within a wrapper `<div>`, but if you add 
`margin: 0 auto;` here, it will center your component grid on the page so that the outside left border isn't against the left side of the screen. Otherwise, this looks great!

Reply:
Will do, thanks!